mmv1/templates/terraform/examples/cloud_run_service_remove_tag.tf.erb

```diff
@@ -0,0 +1,18 @@
+resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
+  name     = "<%= ctx[:vars]['cloudrun_srv'] %>"
+  location = "us-central1"
+
+  template {}
+
+  traffic {
+    percent       = 100
+    revision_name = "cloudrun-srv-green" # This revision needs to already exist
```

Comment:
If we are trying to actually run these tests these will need to exist and be set up

Reply:
Ignoring tests similar to other samples